client/cmdhf15.c

```diff
@@ -1270,6 +1291,8 @@ static int CmdHF15Raw(const char *Cmd) {
             PrintAndLogEx(WARNING, "timeout while waiting for reply.");
         }
     }
+
+    DropField();
```

Comment:
this one is also problematic.  RAW commands allows for the field to stay up...
this would interrupt that process..

Reply:
Maybe we should add a "leave field on" option for it?

Reply:
Yes, I agree, there should be like 14a / 14b raw commands,   the option to restart/leave on/ etc..